src/common/utils/elevation.h

```diff
@@ -83,9 +83,20 @@ namespace
     inline bool GetDesktopAutomationObject(REFIID riid, void** ppv)
     {
         CComPtr<IShellView> spsv;
-        if (!FindDesktopFolderView(IID_PPV_ARGS(&spsv)))
+        
+        // Desktop may not be available on startup
+        auto maxAttempts = 5;
+        for (auto i = 1; !FindDesktopFolderView(IID_PPV_ARGS(&spsv)) && i <= maxAttempts; i++)
```

Comment:
let's move `!FindDesktopFolderView(IID_PPV_ARGS(&spsv))` to the body of `for` loop to improve clarity 🙂